cypress/integration/ete/register_email_sent.spec.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/// <reference types="cypress" />
+import { injectAndCheckAxe } from '../../support/cypress-axe';
+
+describe('Registration email sent page', () => {
+  context('A11y checks', () => {
+    it('has no detectable a11y violations on the registration email sent page', () => {
+      // set encrypted email
+      cy.setCookie('PLAY_SESSION_2', testUserEncryptedEmailCookie, {
+        log: true,
+      });
+      cy.visit(`/register/email-sent`);
+      injectAndCheckAxe();
+    });
+  });
+
+  const testUserEmail = 'test+13411@guardian.com';
+  const testUserEncryptedEmailCookie = Buffer.from(
+    JSON.stringify(testUserEmail),
+  ).toString('base64');
```

Comment:
This is only used in the context of the text below so should live inside that `it` scope. This helps communicate intent and also prevents unwanted naming impacts